app/models/spotlight/exhibit.rb

```diff
@@ -5,6 +5,7 @@ module Spotlight
   ##
   # Spotlight exhibit
   class Exhibit < ActiveRecord::Base
+    class_attribute :themes_selector
```

Comment:
Would happily entertain other ideas for naming this.

Reply:
This could also theoretically go in the engine config, went back and forth and landed here, but not for any reason other than the intention is to have the ability to be exhibit specific.